server/src/migrations/20190906235036_create_categories_table.ts

```diff
@@ -0,0 +1,15 @@
+import * as Knex from "knex";
```

Comment:
what's with this file name, is that normal? A timestmap?

Reply:
That's standard for migrations.  The filename is a combination of the date and a human-readable name